README.md

```diff
@@ -57,6 +57,7 @@ The **Maintained?** column shows:
 |----------|-------------------------|---------------------|-------------|
 | [ARM Juno Board](http://www.arm.com/products/tools/development-boards/versatile-express/juno-arm-development-platform.php) |`PLATFORM=vexpress-juno`| Yes | ![Actively Maintained](documentation/images/green.svg) |
 | [Atmel ATSAMA5D2-XULT Board](http://www.atmel.com/tools/atsama5d2-xult.aspx)|`PLATFORM=sam`| Yes | ![Actively Maintained](documentation/images/green.svg) |
+| [Broadcom ns3](https://www.broadcom.com/)|`PLATFORM=bcm-ns3`| Yes | ![Actively Maintained](documentation/images/green.svg) |
```

Comment:
Can you provide a more specific link? Looking for ns3 at that site doesn't help much unless you know where to navigate.

Reply:
It is not available in public domain. Should I remove the link otherwise.